db/db_block_cache_test.cc

```diff
@@ -1916,14 +1915,12 @@ TEST_P(DBBlockCachePinningTest, TwoLevelDB) {
       ++expected_index_misses;
     }
   }
-  if (unpartitioned_pinning_ == PinningTier::kNone) {
-    ++expected_compression_dict_misses;
-  }
   ASSERT_EQ(expected_filter_misses,
             TestGetTickerCount(options, BLOCK_CACHE_FILTER_MISS));
   ASSERT_EQ(expected_index_misses,
             TestGetTickerCount(options, BLOCK_CACHE_INDEX_MISS));
-  ASSERT_EQ(expected_compression_dict_misses,
+  // for kNoCompression, compression dict is not set/empty
+  ASSERT_EQ(0,
```

Comment:
I can try compression. The reason I didn't do initially was because of this comment https://github.com/facebook/rocksdb/blob/e91263edb9ece1c4cdb2ad83cb93c38ac60bde85/db/db_block_cache_test.cc#L1848-L1851

Do you see this as a problem still?

Reply:
Not a problem. The thing that "seems fairly likely to change over time" has now changed, so I guess we can now update the test accordingly